fedimintd/templates/params.html

```diff
@@ -14,20 +14,20 @@ <h3>Configure Federation</h3>
       <input type="text" class="form-control" id="federation_name" name="federation_name" placeholder="Cypherpunk Federation">
     </div>
     <div class="form-group">
-      <label for="bind_api">Socket address to bind public API</label>
+      <label for="bind_api">API Listen Address</label>
       <input type="text" class="form-control" id="bind_api" name="bind_api" placeholder="127.0.0.1:8173" />
     </div>
     <div class="form-group">
-      <label for="api_url">External Url for public API</label>
-      <input type="text" class="form-control" id="api_url" name="api_url" placeholder="wss://your-website.com" />
+      <label for="api_url">API Public URL</label>
+      <input type="text" class="form-control" id="api_url" name="api_url" placeholder="wss://api.your-website.com" />
     </div>
     <div class="form-group">
-      <label for="bind_p2p">Socket address to bind federation consensus</label>
+      <label for="bind_p2p">P2P Listen Address</label>
       <input type="text" class="form-control" id="bind_p2p" name="bind_p2p" placeholder="127.0.0.1:8174" />
     </div>
     <div class="form-group">
-      <label for="p2p_url">External URL for federation consensus</label>
-      <input type="text" class="form-control" id="p2p_url" name="p2p_url" placeholder="wss://your-website.com" />
+      <label for="p2p_url">P2P Public URL</label>
+      <input type="text" class="form-control" id="p2p_url" name="p2p_url" placeholder="wss://p2p.your-website.com" />
```

Comment:
the p2p url ignores the protocol (here wss) but wss is stil a bit misleadi, since we speak a custom binary protocol

Reply:
This is what API URLs look like currently in `consensus.json`. I just followed that. 
![image](https://user-images.githubusercontent.com/4335621/211394233-a5b76231-37a0-4ac0-9dfd-fb4b4bb65144.png)

Also, `url::Url` doesn't accept it without the protocol prefix https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=ce5d38500da986b050eb361f76625aa7